apps/dev/nuxt/.eslintrc.cjs

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  root: true,
+  parser: "vue-eslint-parser",
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:vue/vue3-recommended",
+    "prettier",
+  ],
+  plugins: ["eslint-plugin-vue", "@typescript-eslint"],
+  ignorePatterns: ["*.cjs", "client.*", "index.*"],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    extraFileExtensions: [".vue"],
+    parser: "@typescript-eslint/parser",
+    sourceType: "module",
+  },
+  env: {
+    browser: true,
+    es2020: true,
+    node: true,
+  },
+}
```

Comment:
Maybe integrate the official ESLint module – https://eslint.nuxt.com/packages/module ?

Reply:
Blog article by the great @antfu – https://nuxt.com/blog/eslint-module

Reply:
Eslint rules and config has been defined for the full monorepo, I don't think the scope of this change would be to change that.